docs/faq/skip_augur_tree.rst

```diff
@@ -0,0 +1,50 @@
+===========================================
+Creating a tree using your own tree builder
+===========================================
+
+The `augur tree` command is a light wrapper around tree building programs such as IQ-TREE, RAxML and FastTree.
+It's possible that the functionality you want isn't available in those programs, or that it is available but that `augur tree` doesn't expose the functionality you need.
+The solution is just to use your own tree building program directly.
+This guide explains the general approach you can use.
+
+
+Exclude sites
+-------------
+
+Augur tree allows you to mask certain problematic sites before tree-building via  `augur tree --exclude-sites <exclude_file> --alignment <alignment>` , where each line in `<exclude_file>` is a 1-based nucleotide position to mask from the alignment. We can recreate this via the following command:
+
+.. code-block:: bash
+  augur mask --sequences <alignment> --mask <exclude_file> --output <masked_alignment>
```

Comment:
This code block isn't rendering in the preview.

Reply:
Fixed in [41e94d6](https://github.com/nextstrain/augur/pull/884/commits/41e94d6eebc76ddbe8d504c1dc05e25134017382)

Reply:
Thanks @joverlee521. I _really_ don't understand rst :(

Reply:
I've been having a hard time with it too. It's been a lot of trial and error 😥

Reply:
To maybe help with understanding, from the [rST primer](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html) provided by Sphinx, [on directives](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#directives):

> The directive content follows after a blank line and is indented relative to the directive start or if options are present, by the same amount as the options.

In general if you're confounded by rST, then I'd really recommend reading through the primer and then the [other rST guides from Sphinx](https://www.sphinx-doc.org/en/master/usage/restructuredtext/index.html).